sdk/netapp/Azure.ResourceManager.NetApp/src/Custom/RestOperations/VaultsRestOperations.cs

```diff
@@ -16,6 +15,8 @@
 
 namespace Azure.ResourceManager.NetApp
 {
+    [EditorBrowsable(EditorBrowsableState.Never)]
+   // [Obsolete("This class is obsolete and will be removed in a future release", false)]
```

Comment:
Since this class is internal, it is hidden all the time, therefore we do not need these attribute, or Obsolete attribute